### ISSUE TITLE
[Feature Request] Ability to select from multiple templates

### ISSUE THREAD

**Issue by dezpixels** [InvoicePlane/InvoicePlane#1050]
It would be nice to be able to select a specific template on time. 
Sometimes we need to use different templates and the way it is set up now is a bit cumbersome. 

## Current Behavior
Currently, if I want to change it, I have to go to the general settings, change it, export the invoice or quote, and then go back to the settings to change it again. 

## Possible Solution
Ideally, you should be able to select the template when quoting, or assign a template to a group of invoices.

**Comment by nielsdrost7** [comment]
> Ideally, you should be able to select the template when quoting
If I'm not mistaking, on the quote create/edit page you can add a select box with all those templates (careful, in that list there are also templates for the invoices)

Select the template from the selectbox, save the quote
In the resulting function handle the selected template (check that function for details)

the field in the database is `group_id`

**Comment by dezpixels** [comment]
The truth is that I am a bit lost with this and no matter how hard I look for detailed information I can't find it. 
I have tried creating a custom field, but I don't know in which way I can make it go associated to the list of templates. 
Can you help me? thanks!

**Comment by J-C-P** [comment]
I agree, being able to select a template by invoice group would be super nice !!

**Comment by nielsdrost7** [comment]
Ok, @dezpixels @J-C-P I've done the following:
- created a branch
- invited you to thr repository of InvoicePlane/InvoicePlane
- if you guys want you can come to Slack. Here's the invite link: https://join.slack.com/t/invoiceplane/shared_invite/zt-2kidwnpkk-yTI7~x9Wi6lE~D0NgMpU6g
I would suggest you guys work on this together.

- edit the `edit invoice` and `create invoice` pages (luckily that's the same file`
- make sure you can select the `invoice_group`
- save the invoice
- see what happens if you save that invoice with the `invoice_group`

When you're on Slack you can easily send short messages to each other.

See of that works out
